### ISSUE TITLE
Cannot change order of subjects using keyboard controls

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1108]
Cannot change the display order of subjects using keyboard navigation in the Project Administrator -> Project -> Subjects view

**Comment by mosefer** [comment]
fixed in #1170
